stempy/electron.cpp

```diff
@@ -229,9 +358,14 @@ std::vector<std::vector<std::pair<int, int>>> electronCount(
         }
```

Comment:
We should do the thresholding in the same pass, otherwise we are still looping through each pixel twice?